Deployment - error when pushing all three apps in manifest-prod-a

```sh
 cf push -f manifests/manifest-prod-a.yml
```

works deploying `calc-prod-a` and `calc-rq-worker-a`, but then fails when it gets to `calc-rqworker-a`:


```txt
Starting app calc-rqworker-a in org oasis / space calc-prod as james.seppi@gsa.gov...
panic: send on closed channel

goroutine 893 [running]:
panic(0x6ea700, 0xc820d55100)
	/usr/local/Cellar/go/1.6.1/libexec/src/runtime/panic.go:464 +0x3e6
github.com/cloudfoundry/cli/cf/api.(*LoggregatorLogsRepository).flushMessageQueue.func1(0xc821dbc690)
	/Users/cffadmin/concourse-work-dir/containers/00002kcgp1v/tmp/build/254d4083/gopath/src/github.com/cloudfoundry/cli/cf/api/log_repository.go:107 +0x40
github.com/cloudfoundry/cli/cf/api.(*Loggregator_SortedMessageQueue).EnumerateAndClear(0xc8201a2a60, 0xc820310748)
	/Users/cffadmin/concourse-work-dir/containers/00002kcgp1v/tmp/build/254d4083/gopath/src/github.com/cloudfoundry/cli/cf/api/loggregator_message_queue.go:46 +0x103
github.com/cloudfoundry/cli/cf/api.(*LoggregatorLogsRepository).flushMessageQueue(0xc8204f30c0, 0xc821af42a0)
	/Users/cffadmin/concourse-work-dir/containers/00002kcgp1v/tmp/build/254d4083/gopath/src/github.com/cloudfoundry/cli/cf/api/log_repository.go:108 +0x45
github.com/cloudfoundry/cli/cf/api.(*LoggregatorLogsRepository).TailLogsFor.func1(0xc8203d5280, 0xc8204f30c0, 0xc821af42a0)
	/Users/cffadmin/concourse-work-dir/containers/00002kcgp1v/tmp/build/254d4083/gopath/src/github.com/cloudfoundry/cli/cf/api/log_repository.go:89 +0x78
created by github.com/cloudfoundry/cli/cf/api.(*LoggregatorLogsRepository).TailLogsFor
	/Users/cffadmin/concourse-work-dir/containers/00002kcgp1v/tmp/build/254d4083/gopath/src/github.com/cloudfoundry/cli/cf/api/log_repository.go:91 +0x383
```

--- a/deploy.md
+++ b/deploy.md
@@ -2,9 +2,10 @@
 
 **Only of interest to 18F team members**
 
-If you don’t already have one, request a Cloud Foundry account in #devops
-in Slack and download the Cloud Foundry CLI according to the instructions here:
-https://docs.18f.gov/getting-started/setup/
+Download the Cloud Foundry CLI according to the instructions here:
+https://docs.18f.gov/getting-started/setup/.
+Make sure you are using a version >= v6.17.1, otherwise pushing multiple apps
+at once might not work.
 
 To start, target the org and space you want to work with. For example, if you wanted to work with the staging space:
 `cf target -o oasis -s calc-dev`

--- a/deploy.md
+++ b/deploy.md
@@ -3,7 +3,7 @@
 **Only of interest to 18F team members**
 
 Download the Cloud Foundry CLI according to the instructions here:
-https://docs.18f.gov/getting-started/setup/.
+https://docs.cloud.gov/getting-started/setup/.
 Make sure you are using a version >= v6.17.1, otherwise pushing multiple apps
 at once might not work.
 

